Option for Deterministic ZipInfo

zommuter opened the pull request:
Adds a `--deterministic` switch to `rezip.py`. All `ZipInfo` except for the filename is reverted to a default value, so metadata changes (especially timestamps) won't make git consider the file changed. The downside is of course, said metadata _could_ be important, therefore it's optional.

zommuter committed:
make deterministic info optional

costerwi commented on rezip.py:
```diff
@@ -31,7 +36,13 @@
+            if args.deterministic:
+                deterministic_info = ZipInfo(info.filename)
+                deterministic_info.create_system = 0  # everything else is fixed
+                newinfo = deterministic_info
```
Thanks for the contribution.
Can we eliminate the "deterministic_info" variable by using "newinfo" at line 40?

zommuter replied:
Sure thing, don't know why I left it in there.

costerwi reviewed (changes requested):
Thanks!

zommuter commented:
Ok, I removed the unnecessary `deterministic_info` (don't know why I actually had that one). I squashed the commit, so your review comment and my reply got removed...
